code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -534,6 +534,16 @@
 		/obj/item/stock_parts/capacitor = 1)
 	def_components = list(/obj/item/stock_parts/cell = /obj/item/stock_parts/cell/high/empty)
 
+/obj/item/circuitboard/machine/rad_collector
+	name = "Radiation Collector (Machine Board)"
+	build_path = /obj/machinery/power/rad_collector
+	req_components = list(
+		/obj/item/stack/cable_coil = 5,
+		/obj/item/stock_parts/matter_bin = 1,
+		/obj/item/stack/sheet/plasmarglass = 1,
+		/obj/item/stock_parts/manipulator = 1)
```

Comment:
I'm very hesitant to allow the construction of these at all but it doesn't make much sense that they can't be.

Err on the side of expensive for now, make it require a second glass and a capacitor.